src/formElementPlugins/datePicker/src/ApplicationView.tsx

```diff
@@ -35,13 +37,15 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   const [selectedDate, setSelectedDate] = useState<SelectedDateRange>(
     fromDateSaved(currentResponse?.date)
   )
+  const {
+    selectedLanguage: { locale },
+  } = useLanguageProvider()
   const { isEditable } = element
   const {
     label,
     description,
     default: defaultDate,
     allowRange = Array.isArray(defaultDate),
-    locale = 'en-US',
```

Comment:
We don't want to remove this -- the locale should still be over-ride-able from the element configuration.

Just change the default value instead:
```
locale = selectedLanguage.locale
```
(And remove the nested destructure above to avoid name duplication)